### PR TITLE
Add Instanbul test coverage to Codeclimate

### PR DESCRIPTION
**Pull request by sleepyfox**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@ package-lock.json
 node_modules
 *~
 \#*\#
+.nyc_output
```
Comment (rifkinni) [inline .gitignore:5]: trivial: newline

File: .github/workflows/nodejs.yml [hunk 2]
```diff
@@ -18,6 +18,12 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
+        env:
+          CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
```
Comment (rifkinni) [inline .github/workflows/nodejs.yml:22]: where is this environment variable needed/used?

Comment (sleepyfox) [reply .github/workflows/nodejs.yml:22]: The env var is used by the CodeClimate test reporter.

File: .gitignore (hunk 1 above)
Comment (sleepyfox) [reply .gitignore:5]: Generated file.

Comment (sleepyfox) [reply .gitignore]: Fixed.

**Pull request merged**